wgsl/index.bs

```diff
@@ -2971,10 +2971,10 @@ and third clauses and in the body of the `for` statement.
   <thead>
     <tr><th>Precondition<th>Conclusion<th>Notes
   </thead>
-  <tr><td>*e* : bool<td>`bool(e)` : bool<td>Identity.<br>In the SPIR-V translation, the ID of this expression reuses the ID of the operand.
```

Comment:
For some reason I expect
-  a space before the colon in type assertions that are in a mathematical context, e.g. in the preconditions column of the type tables
- no space before the colon in actual program text.

I know that's not consistent.  But it's a weak preference, and it's probably much better to make the math look more like the program text.  After, all that was the point of using that notation in the programming language.

So all this is good.  I'm just noting my 5% preference.